### DISCUSSION TITLE
Sideways bars

### DISCUSSION THREAD

**Discussion by LimaVsC** [zauberzeug/nicegui#4082] [Q&A] [answered]
### Question

Is there a way to rotate the content of a tab by 90°, like in the image below?

![asd](https://github.com/user-attachments/assets/5aeb4bac-f524-446d-8b52-6611041d09dc)

**Comment by falkoschindler** [comment, marked as answer]
Hi @LimaVsC,

Using the "content-class" prop allows to customize the appearance of tabs, e.g. using the Tailwind class "rotate-270":

```py
with ui.row():
    with ui.tabs().props('vertical') as tabs:
        one = ui.tab('One').props('content-class=rotate-270')
        two = ui.tab('Two').props('content-class=rotate-270')
    with ui.tab_panels(tabs, value=one).props('vertical'):
        with ui.tab_panel(one):
            ui.label('First tab')
        with ui.tab_panel(two):
            ui.label('Second tab')
```
<img width="156" alt="Screenshot 2024-12-10 at 07 28 11" src="https://github.com/user-attachments/assets/a679b9e5-92fd-4431-82de-3d55077715c7">

**Reply by LimaVsC** [reply]
Thanks!
